woocommerce/woocommerce-android
# Product creation using package photo: Title & description generation

This PR adds the product title and description generation using the scanned keywords from a package photo. 

<img src="https://github.com/woocommerce/woocommerce-android/assets/1522856/187dc618-60bf-4553-97ba-3d7277413acc" width="350" />

**To test:**
1. Go to Products
2. Tap on + FAB
3. Select the AI option
4. Tap on Use package photo button
5. Use the media library to select a picture of a product with some text
6. Notice the image is loaded and a list of keywords from the image is displayed
7. Notice the loading state is shown and the keywords section is disabled
8. Notice that after a few moments, the product title and description is shown
9. Try unchecking all the keywords
10. Notice the Regenerate button gets disabled
11. Repeat steps 5-7 with an image that contains no text
12. Notice a message is shown that informs the user that no text was found
